uber/models.py

```diff
@@ -693,53 +693,6 @@ def shift_badges(self, badge_type, badge_num, *, until=None, **direction):
                 a.badge_num += shift
             return True
 
-        def change_badge(self, attendee, badge_type, badge_num=None):
```

Comment:
Yup, this was an oops on my part during the refactor last year. In order to work on stuff incrementally, I left this function as-is and made a new function to replace it called `update_badge`. I finished the refactor, but just totally forgot to remove the obsoleted function!

Confusingly, there's a `change_badge` function inside our tests -- I probably did a quick review of things and thought that calls to that function were really calls to this function. They were not.